app/processing/delinked-statement/delinkedCalculation/get-delinked.js

```diff
@@ -0,0 +1,10 @@
+const getDelinkedByCalculationId = require('./get-delinked-by-calculation-id')
+const validateDelinked = require('./validate-delinked')
+
+const getDelinked = async (calculationId) => {
+  const delinked = await getDelinkedByCalculationId(calculationId)
+  return validateDelinked(delinked, calculationId)
+}
+console.log('validated-delinked in get-delinked', validateDelinked)

```

Comment:
Same as above, any need to log the functions

Reply:
The reverse of above, because we are sending a transformed field, this is to show in logs that it's been set back.